warehouse/upload.go

```diff
@@ -764,46 +752,50 @@ func (job *UploadJobT) loadIdentityTables(populateHistoricIdentities bool) (load
 	identityTables := []string{job.identityMergeRulesTableName(), job.identityMappingsTableName()}
 
 	var alteredSchema bool
```

Comment:
In above line `if generated, _ := job.areIdentityTablesLoadFilesGenerated(); !generated`
should we even do the `resolveIdentities()` part? Maybe we can skip at the start of the func only if something in prev uploads has failed?

Reply:
Done